packages/app/src/sandbox/eval/transpilers/svelte/svelte-worker.js

```diff
@@ -80,11 +124,20 @@ function getV1Code(code, path) {
 }
 
 self.addEventListener('message', event => {
-  const { code, path, isV2 } = event.data;
+  const { code, path, version } = event.data;
+  let versionCode = '';
+
+  if (semver.satisfies(version, '1.x')) {
+    versionCode = getV1Code(code, path);
+  }
+  if (semver.satisfies(version, '2.x')) {
+    versionCode = getV2Code(code, path);
+  }
+  if (semver.satisfies(version, '3.x')) {
+    versionCode = getV3Code(code, path);
```

Comment:
Could these `get___` functions receive the version string so that the `unpkg` URLs can use the provided version?

```js
function getVXCode(code, path, version) {
  self.importScripts([`https://unpkg.com/svelte@${version}/compiler/svelte.js`]);
```

Reply:
This is an awesome idea! Done!